src/Onboarding/Config/CauseTypes.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+	'advocacy'      => __( 'Advocacy', 'give' ),
+	'education'     => __( 'Educational', 'give' ),
```

Comment:
Little thing, but curious about using "education" when the label is "educational". Just trying to keep consistent with the other option values (which simply reflect their label).

Reply:
I think this is me passive-aggressively suggesting to use "Education" as the label, instead of "Educational"...

I meant to raise the question of which it should be, but forgot.

Reply:
Haha, makes sense - yeah I'm sorta thrown by the extra syllable in "educational." Maybe worth checking with marketing on.

Reply:
@henryholtgeerts updated.